src/App.js

```diff
@@ -233,12 +233,10 @@ function App() {
         {hymns.map((hymn, index) => (
           <div
             className="cantorTab001"
-            style={textStyle}
+            style={{paddingRight: "0"}}
             key={index}
             onClick={() => {
-              if (expandedHymns === index) {
-                setExpandedHymns(-1);
-              } else {
+              if (expandedHymns !== index) {
```

Comment:
if a hymn is expanded dont unexpand onclick (this is how I got the buttons to be clickable). now the hymns will unexpand when a different hymns is selected